src/lib/service-bus/get-connector.js

```diff
@@ -0,0 +1,17 @@
+import { AzureServiceBus } from './AzureServiceBus.js';
+import { MessageSender } from 'ffc-messaging';
+
+/**
+ * Returns an instance of a ServiceBus, depending on the environment.
+ * In non-local environments, it returns a MessageSender configured with the contact details queue.
+ * In local environment, it returns a Redis instance connected to the 'redis' host on port 6379.
+ * @param app
+ * @returns {object} Message sender
+ */
+export function getConnecter(app) {
+  if (app.environment === 'local') {
+    return new AzureServiceBus(app.serviceBus.local.serviceBusConnectionString);
```

Comment:
Do we need different connectors? I thought the new emulator would enable us to use the same connector taking a connection string? If we have just one connector, we can have more confidence that the developer tests prove the deployed solution works.

Reply:
I didn't know that this solution works in live environments. I thought it was just a local option and that we would still need the ffc messaging solution for deployment.